### DISCUSSION TITLE
Некорректно отрабатывает перелистывание страниц

### DISCUSSION THREAD

**Discussion by smplseven** [totumonline/totum-ru-issues-and-discussions#428] [ОШБК!] [answered]
**Описание:**
При использовании пагинации и включенной функции автообновления страницы ,пролистывание строк работает неккоректно, а именно:

При нажатии на переход к следующей страницы, система  снова откидывает на предыдущую страницу

При нажатии на переход к следующей странице система может показать следующий номер страницы, но таблица будет отображать предыдущую страницу

https://github.com/totumonline/totum-ru-issues-and-discussions/assets/113604604/f6d02997-6d25-4af5-82ea-2265e325b7c5

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Может запретить вообще листание для режима автообновления — было бы хорошо мне кажется.

**Reply by ttm-alexeypolunin** [reply]
Не повторяется у нас. Пока вопрос закрыт
